src/NewsletterEpic/index.stories.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { ReactElement } from 'react';
+import { BrazeEndOfArticleComponent } from '../BrazeEndOfArticleComponent';
+import {
+    StorybookWrapper,
+    mockSubscribe,
+    mockFetchEmail,
+    mockButtonClick,
+    mockComponentEvent,
+} from '../utils/StorybookWrapper';
+import { knobsData } from '../utils/knobsData';
+import { coreArgTypes, ophanComponentIdArgType } from '../storybookCommon/argTypes';
+import { BrazeMessageProps } from '.';
+import { grid, withGrid } from '../../.storybook/grid/withGrid';
+
+export default {
+    component: 'NewsletterEpic',
+    title: 'EndOfArticle/NewsletterEpic',
+    decorators: [withGrid],
+    parameters: {
+        grid: {
+            disable: false,
+        },
+    },
+    argTypes: {
+        ...coreArgTypes,
+        ...ophanComponentIdArgType,
+        newsletterId: {
+            name: 'newsletterId',
+            type: { name: 'string', required: true },
+            description: 'The newsletter Id value',
+        },
+        imageUrl: {
+            name: 'imageUrl (use Grid image picker)',
+            type: { name: 'string', required: true },
+            control: null,
+            description:
+                'i.guim.co.uk URL for the epic image. Use the Grid image picker to select this.',
+        },
+        header: {
+            name: 'header',
+            type: { name: 'string', required: true },
+            description: 'Header text',
+        },
+        frequency: {
+            name: 'frequency',
+            type: { name: 'string', required: true },
+            description: 'Text description of how often the email is sent',
+        },
+        paragraph1: {
+            name: 'paragraph1',
+            type: { name: 'string', required: true },
+            description: 'First paragraph',
+        },
+        paragraph2: {
+            name: 'paragraph2',
+            type: { name: 'string', required: false },
+            description: 'Second paragraph',
+        },
+    },
+};
+
+const StoryTemplate = (
+    args: BrazeMessageProps & {
+        componentName: string;
+        newsletterId: string;
+    },
+): ReactElement | null => {
+    const imageUrl = grid(
```

Comment:
Does the grid plugin generate images which are appropriately sized for the Braze epic? (I _think_ this is the first use case for it in the epic?)